Robot.java_03_22_17

```diff
@@ -1,518 +1 @@
-		right.setInverted(true);
-		this.left = new Good_DriveSide(KpL, Ki, Kd, Kf, leftEncoder, leftSpark);
-
-		
-		driveTrain = new RobotDrive(left,right);
-	//	driveTrain = new RobotDrive(leftSpark, rightSpark);
-
-		/*
-		 * String autoSelected = SmartDashboard.getString("Auto Selector",
-		 * "Default"); switch(autoSelected) { case "My Auto": autonomousCommand
-		 * = new MyAutoCommand(); break; case "Default Auto": default:
-		 * autonomousCommand = new ExampleCommand(); break; }
-		 */
-		//autonomousCommand = new MoveForward();
-		// schedule the autonomous command (example)
-		if (autonomousCommand != null)
-			autonomousCommand.start();
-		
-
-		myTimer.reset();
-		myTimer.start();
-		
-		
-	//leftEncoder.reset();
-		gyro.calibrate();
-		//System.out.println("gyro calibration: " + gyro.getAngle());
-		
-	}
-
-	/**
-	 * This function is called periodically during autonomous
-	 */
-	@Override
-	public void autonomousPeriodic() {
-		
-		Scheduler.getInstance().run();
-	
-		//	System.out.println("encoder count: "+ ((Good_DriveSide) left).getEncoderCount());
-			//QUESTION: if i made my own method inside good_driveside, which implements speedController
-			//do I have to cast left as a goodDriveside instead of speedcontroller?
-		/*	
-		 * type in the number of feet you want to go, it'll go until it reaches that distance*/
-/*		System.out.print("right rate: " + rightEncoder.getRate());
-		System.out.println("\t\t\tright encoder distance	" + rightEncoder.getDistance());
-		System.out.print("left rate: " + leftEncoder.getRate());
-		System.out.println("\t\t\tleft encoder distance	" + leftEncoder.getDistance());
-		//autoDriveStraight(1.5);		
-	if (leftEncoder.getDistance() < feetToEncUnits(5) && rightEncoder.getDistance() < feetToEncUnits(5)) {
-			//rotations: X * 2400 / 360 
-			//feet: 
-			left.set(.3);
-			right.set
-			\\System.out.println("        !        ");
-			
-		}
-		else {
-			System.out.println("    it went 5 feet    ");
-			left.set(0);
-			right.set(0);
-		}  */
-	/*	if (myTimer.get() < (3)) {
-			//rotations: X * 2400 / 360 
-			//feet: 
-			left.set(.1);
-			right.set(.1);
-			System.out.println("        !        ");
-			
-		}
-		else {
-			System.out.println("    3 seconds   ");
-			left.set(0);
-			right.set(0);
-		} 
-		*/
-		
-		//middle lane 
-		System.out.println(state);
-		if (state.equals("middleforward1")) {//midlane
-		//	moveForward(9.00);
-			moveForwardSlow(5);
-			if (finishedMoving(5)) 
-				state = "stop";
-		}
-		
-		//right lane 
-		else if (state.equals("rightforward1")) {
-			moveForward(9.5167);
-			//10.83 feet = 129.96 inches
-			//9.5167 feet = 114.3 inches
-			//have to calculate again with the length of the robot and the distance from the center
-			//from left and right positions not the middle
-			if (finishedMoving(9.5167))
-				state = "rightturn1";
-		}
-		else if (state.equals("rightturn1")) {
-			//this is a right turn
-			changeAngle(10); 
-			if (finishedTurning(10))
```

Comment:
A small nit, but it's always a good idea to use braces explicitly in if statements even if it's one line. It's too easy to want to add something later (like maybe a print statement) and then only one line is actually in the print statement.